test/logger_test.rb

```diff
@@ -446,7 +446,18 @@ class TestLogger < Hanami::Logger; end
               class TestLogger < Hanami::Logger; end
               TestLogger.new.info(['foo'])
             end
-          output.must_equal "app=Hanami severity=INFO time=1988-09-01 00:00:00 UTC message=[\"foo\"]\n"
+          output.must_equal "app=Hanami severity=INFO time=1988-09-01T00:00:00+00:00 message=\'[\"foo\"]\'\n"
+        end
+      end
+
+      it 'wraps messages with spaces into single quotes' do
+        stub_time_now do
+          output =
+            stub_stdout_constant do
+              class TestLogger < Hanami::Logger; end
+              TestLogger.new.info('This is a message with spaces')
```

Comment:
@AlfonsoUceda What happens when a message with `'` is passed? We should escape it, right now it returns:

```ruby
TestLogger.new.info("This isn't a message without '")
```

```
"app=Hanami severity=INFO time=1988-09-01T00:00:00+00:00 message='This isn't a message without ''
 "
```